pymc/sampling.py

```diff
@@ -2125,18 +2125,18 @@ def draw(
 
     Parameters
     ----------
-    vars
+    vars : tensor_like or iterable of tensor_like
```

Comment:
I made a mistake here before, tensor_like includes things that can be converted to aesara tensorvariable like lists or numpy arrays, that is not the case here. 

```suggestion
    vars : Variable or iterable of Variable
```